installing.rst

```diff
@@ -28,137 +37,148 @@ For this to work, you will need to install `homebrew <http://brew.sh/>`__. If yo
 Linux
 -----
 
-`Download <http://gauge.org/get-started.html>`__ the zip file. Choose the
-archive file appropriate for your installation. And run the command
-below to install Gauge.
+On Debian, Ubuntu
```

Comment:
all steps can be clubbed into one single code block. The descriptions can also be clubbed on top.